frontend/src/components/pages/GlobalForms/GlobalFormsPage.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {Text, Container} from '@chakra-ui/react'
+import AdminAPIClient from "../../../APIClients/AdminAPIClient";
+import { ClauseInfo } from "../../../types/AdminTypes";
+import WaiversTab from "./WaiversTab";
+
+const GlobalFormsPage = (): React.ReactElement => {
+
+  const [waiver, setWaiver] = React.useState([]as ClauseInfo[]);
```

Comment:
If you're using waiver as the state, I would recommend making this state of the type `WaiverResponse` 
This would look like:
```
const [waiver, setWaiver] = React.useState<WaiverResponse>()
```

Then instead of setting the state to the set of clauses, you would just set the state to whatever your APIClient getWaiver function returns since you are enforcing the return type of that function to `WaiverResponse`

Reply:
If you want to use the clauses only, I would suggest renaming the state to waiverClauses so it's a bit more clear